### PR TITLE
install sentry.io node.js sdk 

### PR DESCRIPTION
**Pull request by PaulApivat**
**What?**

-  Preparing sentry.io installation by adding a script to bounty-board/packages/react-app/src/pages/_app.tsx.
- Add @sentry/node as a dependency with yarn add
- create .env file in root directory to store SENTRY_DSN

### CODE REVIEW

**Comment by PaulApivat** [comment]
Hey @Behold3th here's the documentation for installing Sentry with NodeJS https://docs.sentry.io/platforms/node/
I used the CommonJS pattern.

I believe the error comes from the `foo()` in `setTimeout()`.

File: packages/react-app/src/pages/_app.tsx [hunk 1]
```diff
@@ -14,10 +14,46 @@ import '../styles/css/nprogress.css';
 import { CustomerContext, getCustomerFromBountyId, setCustomerFromLocalStorage } from '../context/CustomerContext';
 import { BANKLESS } from '../constants/Bankless';
 
+
 Router.events.on('routeChangeStart', () => NProgress.start());
 Router.events.on('routeChangeComplete', () => NProgress.done());
 Router.events.on('routeChangeError', () => NProgress.done());
 
+/* ---------Start Sentry.io installation--------- */
```
Comment (jordaniza) [inline packages/react-app/src/pages/_app.tsx:22]: Recommend the logic be moved to a separate file, then imported into the app.tsx file

**Comment by jordaniza** [comment]
@PaulApivat added some review comments. No major issues but I'd take out the intentional error if the environment is production. @Behold3th what are the vercel logs saying?

**Comment by PaulApivat** [comment]
Update: 

1. I followed the NextJS installation guide for sentry.io: https://docs.sentry.io/platforms/javascript/guides/nextjs/
2. The following config files were create automatically as boiler plate inside react-app: a) sentry.client.config.js, sentry.server.config.js, sentry.properties and next.config.js
3. There were multiple linting errors across a couple flies were fixed with `yarn lint filename.js --fix` thx for that @jordaniza 

Once this PR gets reviewed and merged, the next step is to "add Sentry" to Vercel with this guide: https://docs.sentry.io/product/integrations/deployment/vercel/

Will reach out to @Behold3th

**Comment by PaulApivat** [comment]
Thanks  @jordaniza , made all suggested changes. 

Had to tweak the code a bit for it to compile successfully:

`const debugString = process.env.SENTRY_DEBUG;`
`const debug = debugString === "true";`

**Comment by jordaniza** [comment]
Will reopen this workstream when it is active

**Pull request closed without merging**